media/js/firefox/family-index.js

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+;(function($, _gaq, site) {
+    'use strict';
+
+    var $html = $(document.documentElement);
+    var $downloadbar = $('#conditional-download-bar');
+
+    // Open external links in new window/tab
+    $('.product a[rel="external"]').attr('target', '_blank');
+
+    // Dismiss the conditional download bar
+    $downloadbar.on('click', '.btn-close', function() {
+        $downloadbar.animate({
+            top: '-' + ($downloadbar.height() + 10)
+        }, 350, function() {
+            $downloadbar.fadeOut().remove();
+        });
+    });
+
+    // Check Firefox version
+    if (isFirefox()) {
+        // data-latest-firefox includes point release information
+        var latestFirefoxVersionFull = $html.attr('data-latest-firefox');
+
+        // get latest full version (no point release info) for initial check
+        var latestFirefoxVersion = parseInt(latestFirefoxVersionFull.split('.')[0], 10);
+
+        if (isFirefoxUpToDate(latestFirefoxVersion + '')) {
+            $html.addClass('firefox-latest');
+        } else {
+            $html.addClass('firefox-old');
+        }
+    }
+
+    // Track product clicks
+    $('.product-list').on('click', 'a', function(e) {
+        var newTab = (this.target === '_blank' || e.metaKey || e.ctrlKey);
+        var href = this.href;
+        var product = this.data('product');
+        var callback = function() {
+            window.location = href;
+        };
+
+        if (newTab) {
+            trackGenericLink('/firefox/products/ Interactions', product);
```

Comment:
Oops. I had made a bunch of edits I wasn't ready to commit so I stashed them while I switched to another branch, then forgot to reapply the stash when I switched back.